experimental/generation/generator/packages/library/templates/standard/ageEntity-add.dialog.lg

```diff
@@ -4,4 +4,9 @@
 - ${addFilename('age')}
 
 # template
-- ${addWithUnits('age')}
\ No newline at end of file
+- IF: ${propertySchema.minimum || propertySchema.maximum}
+    - ${addWithUnitsAndMinMax('age')}
+- ELSEIF: ${propertySchema.items.minimum || propertySchema.items.maximum}
+    - ${addItemsWithUnitsAndMinMax('age')}
+- ELSE:
+    - ${addWithUnits('age')}
```

Comment:
Why not have a single addWithUnits and move these tests inside that template to conditionally generate min/max?  That way you don't have to replicate for each place the template is called.